### PR TITLE
Export-Migration

### PR DESCRIPTION
**Pull request by splatteredbits**
Created an `Export-Migration` function that exports a SQL Server database as a Rivet migration.

### CODE REVIEW

**Comment by splatteredbits** [comment]
@webmd-health-services/devops Removed updating Whiskey from this PR. There are several tests that were migrated from Blade to Pester and their files were renamed accordingly. I wish Github showed/understood renames.

File: Rivet/Functions/Columns/New-Column.ps1 [hunk 1]
```diff
@@ -121,44 +121,138 @@ function New-Column
...
+
         [Parameter(ParameterSetName='Nullable')]
         [Switch]
         # Optimizes the column storage for null values. Cannot be used with the `NotNull` switch.
         $Sparse,
 
         [Parameter(ParameterSetName='NotNull')]
         [Switch]
-        # Makes the column not nullable.  Canno be used with the `Sparse` switch.
+        # Makes the column not nullable.  Cannor be used with the `Sparse` switch.
```
Comment (decoyjoe) [inline Rivet/Functions/Columns/New-Column.ps1]: Typo: `Cannor`

File: Rivet/Functions/Columns/New-Column.ps1 [hunk 2]
```diff
@@ -121,44 +121,138 @@ function New-Column
...
+        [Parameter(Mandatory,Position=1)]
         [string]
-        # The datatype of the new column, including precision/scale/size specifiers.
+        # The datatype of the new column. Scale/size/precision clause is optional. 
         $DataType,
 
+        [Parameter(ParameterSetName='Nullable')]
+        [Parameter(ParameterSetName='NotNull')]
+        [Switch]
+        # The size/length of the column.
```
Comment (decoyjoe) [inline Rivet/Functions/Columns/New-Column.ps1]: This parameter doc is the exact same as `$Size`, should it rather read something like `Maximum column character length`?

File: Rivet/Rivet.psd1 [hunk 3]
```diff
@@ -197,8 +198,9 @@ Rivet is a database migration/change management/versioning tool inspired by Ruby
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
-* Adding support for running Rivet under Windows PowerShell 5.1 and PowerShell Core.
-* You can now specify the order Rivet should apply migrations to multiple databases with the "DatabaseOrder" setting in your rivet.json file. It should be set to a list of databases and Rivet will apply migrations to databases in that order. See `help about_Rivet_Configuration` for more information.
+* Created `Export-Migration` function for exporting database objects as Rivet migrations.
+* Rivet can now add XML columns that don't have schema associated with them.
+* New-Column can now be used to create columns on tables that have custom size specifications, are rowguidcol, are identities, custom collations, and are file stream.
```
Comment (decoyjoe) [inline Rivet/Rivet.psd1]: Typo: "New-Column" should have backticks around it.

**Review by decoyjoe — CHANGES_REQUESTED** [review]

**Comment by splatteredbits** [comment]
@DecoyJoe Ready for another review.

**Review by decoyjoe — APPROVED** [review]

**Pull request merged**